tests/test_sites_toolbox.py

```diff
@@ -64,4 +64,13 @@ def test_get_site_group_details(db_session):
 
   assert site_group_sites == [{"site_uuid": str(site.site_uuid), "client_site_id": str(site.client_site_id)}for site in site_group.sites]
   assert site_group_users == [user.email for user in site_group.users]
-  
\ No newline at end of file
+
+def test_change_user_site_group(db_session):
+  """Test the change user site group function"""
+  site_group = make_site_group(db_session=db_session)
+  user = make_user(db_session=db_session, email="test_user@gmail.com", site_group=site_group)
+  site_group2 = make_site_group(db_session=db_session)
```

Comment:
You can tell it what site group to name

https://github.com/openclimatefix/pv-site-datamodel/blob/main/pvsite_datamodel/write/user_and_site.py#L29

`site_group2 = make_site_group(db_session=db_session, site_group_name='test_2')`